src/main/java/bio/terra/cli/command/workspace/Clone.java

```diff
@@ -36,9 +37,17 @@ protected void execute() {
     workspaceOption.overrideIfSpecified();
     Workspace sourceWorkspace = Context.requireWorkspace();
 
+    String spendProfile =
+        Context.getServer().getUserManagerUri() != null
+            ? SpendProfileManagerService.fromContext().getDefaultSpendProfile(null)
+            : "wm-default-spend-profile";
```

Comment:
you have this string hardcoded in various places, but there is a function to get this: Context.getServer().getWsmDefaultSpendProfile())

or better yet pass null and have the workspace service do this in one spot?